pkg/maps/lbmap/affinity.go

```diff
@@ -103,7 +103,9 @@ func (k *AffinityMatchKey) GetKeyPtr() unsafe.Pointer { return unsafe.Pointer(k)
 func (v *AffinityMatchValue) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(v) }
 
 // String converts the key into a human readable string format
-func (k *AffinityMatchKey) String() string { return fmt.Sprintf("%d %d", k.BackendID, k.RevNATID) }
+func (k *AffinityMatchKey) String() string {
```

Comment:
Some of the structs implement the `ToNetwork()` method. Have you considered adding `ToHost()` method? Same applies to all other changed `String()` methods.

Reply:
Encapsulating the conversion byte order to host is a good idea;
In addition, during the debugging process, I feel that the readability of the dumped information is not very friendly; lack of prompt information to help users understand the output;
I plan to make some improvements and optimizations for these issues in the next few days.